onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3522,25 +3541,42 @@ def aten_native_group_norm_backward(
 @torch_op("aten::native_layer_norm", trace_only=True)
 def aten_native_layer_norm(
     input: TReal,
-    normalized_shape: INT64,
+    normalized_shape: Sequence[int],
```

Comment:
Why can we don't use this directly? otherwise

```suggestion
    normalized_shape: Sized,
```